101-XSS-Attack-Prevention/nested/microsoft.sql/servers.databases.json

```diff
@@ -0,0 +1,176 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "sqlServerName": {
+      "type": "string"
+    },
+    "databaseName": {
+      "type": "string",
+      "defaultValue": "contosodb"
+    },
+    "location": {
+      "type": "string"
+    },
+    "edition": {
+      "type": "string",
+      "defaultValue": "Basic",
+      "allowedValues": [
+        "Web",
+        "Business",
+        "Basic",
+        "Standard",
+        "Premium",
+        "Free",
+        "Stretch",
+        "DataWarehouse",
+        "System",
+        "System2"
+      ]
+    },
+    "tags": {
+      "type": "object"
+    },
+    "omsWorkspaceResourceId": {
+      "type": "string"
+    },
+    "storageKeyType": {
+      "type": "string",
+      "defaultValue": "SharedAccessKey"
+    },
+    "administratorLogin": {
+      "type": "string"
+    },
+    "administratorLoginPassword": {
+      "type": "securestring"
+    },
+    "bacpacuri": {
+      "type": "string"
+    },
+    "logsRetentionInDays": {
+      "type": "int",
+      "defaultValue": 0,
+      "minValue": 0,
+      "maxValue": 365,
+      "metadata": {
+        "description": "Specifies the number of days that logs are gonna be kept. If you do not want to apply any retention policy and retain data forever, set value to 0."
+      }
+    }
+  },
+  "resources": [
+    {
+      "name": "[concat(parameters('sqlServerName'),'/',parameters('databaseName'))]",
+      "type": "Microsoft.Sql/servers/databases",
+      "apiVersion": "2017-10-01-preview",
+      "tags": "[parameters('tags')]",
+      "location": "[parameters('location')]",
+      "properties": {
+        "createMode": "Default",
+        "edition": "[parameters('edition')]"
+      },
+      "resources": [
+        {
+          "name": "Import",
+          "type": "extensions",
+          "apiVersion": "2014-04-01-preview",
+          "dependsOn": [
+            "[resourceId('Microsoft.Sql/servers/databases',parameters('sqlServerName'),parameters('databaseName'))]"
+          ],
+          "properties": {
+            "storageKey": "?",
+            "storageKeyType": "[parameters('storageKeyType')]",
```

Comment:
use literal value here and remove the param

Reply:
removed the parameter